packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -516,3 +524,118 @@ export function useOnMount(onMount?: TLOnMountHandler) {
 		if (editor) return onMountEvent?.(editor)
 	}, [editor, onMountEvent])
 }
+
+const WATERMARK_SRC = `url('data:image/svg+xml;utf8,${encodeURIComponent(watermarkDesktopSvg)}') center 100% / 100% no-repeat`
+
+const Watermark = React.memo(() => {
+	const events = useCanvasEvents()
+
+	const ref = useRef<HTMLAnchorElement>(null)
+	useLayoutEffect(() => {
+		// eslint-disable-next-line deprecation/deprecation
+		if (ref?.current) ref.current.style.webkitMask = WATERMARK_SRC
```

Comment:
can't just do `webkitMask` - we have to also do `mask`

Reply:
I see this is resolved - is it because the other place we do `webkitMask` in the code has this comment?
```

			// HACK: Fix for <https://linear.app/tldraw/issue/TLD-1700/dragging-around-with-the-handtool-makes-lots-of-requests-for-icons>
			// It seems that passing `WebkitMask` to react will cause a render on each call, no idea why... but this appears to be the fix.
			// @ts-ignore
			// eslint-disable-next-line deprecation/deprecation
```

prbly worth adding the comment here, too - since it seems this is trying to work around something.